.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   macos_Build:
     name: Mac OS Build
     # The type of runner that the job will run on
-    runs-on: macos-latest
+    runs-on: macos-10.15
```

Comment:
I'd say this is an appropriate fix to get the Github Actions to run the workflow. We don't necessarily need to be running on the latest MacOS (for now).